recipes/7bitconf/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
For new recipes test_v1_package can be dropped, no longer necessary.

Reply:
test_v1_package is now removed.